Sprint-1/destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,34 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+function schoolName(hogwarts) {
+  for (let i = 0; i < hogwarts.length; i++) {
```

Comment:
`for` loop is a legit approach. But now we have better ways to iterate over each element in the array in Javascript. Can you apply alternative method instead of `for` loop here?